test/integration/controlplane/apiserver_identity_test.go

```diff
@@ -84,7 +85,7 @@ func TestCreateLeaseOnStart(t *testing.T) {
 		leases, err := kubeclient.
 			CoordinationV1().
 			Leases(metav1.NamespaceSystem).
-			List(context.TODO(), metav1.ListOptions{LabelSelector: controlplane.KubeAPIServerIdentityLeaseLabelSelector})
+			List(context.TODO(), metav1.ListOptions{LabelSelector: controlplaneapiserver.IdentityLeaseComponentLabelKey + "=" + controlplane.KubeAPIServer})
```

Comment:
Just for learning, why this changes?

Reply:
`KubeAPIServerIdentityLeaseLabelSelector` was a strange constant, although the components were defined just next to it. Also we want other component names than `kube-apiserver` for the generic case.